src/group_theory/order_of_element.lean

```diff
@@ -449,7 +449,7 @@ begin
   obtain ⟨k, rfl⟩ := subgroup.mem_zpowers_iff.mp hx,
   rw [← mul_action.to_perm_apply, ← mul_action.to_perm_hom_apply,
     monoid_hom.map_zpow _ y k, mul_action.to_perm_hom_apply],
-  exact equiv.is_fixed_pt.zpow hs k,
+  exact function.is_fixed_pt.perm_zpow hs k,
```

Comment:
I thought the whole point was that you could now use dot notation!

Reply:
Yes, but `hs` is not syntactically `is_fixed_pt`. Blame Oliver 🙃